From 0ae67c59e8d4e4502d86a7c2b89436799bbdd4b1 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Fri, 12 Jul 2013 15:48:57 +0900
Subject: [PATCH 0833/1083] Revert "ARM: shmobile: Initial r8a7790 SoC support"

This reverts commit 1c48865ffdae40d50470d05ef59513f38251710c.

This adds a second, incomplete, r8a7790 Kconfig entry.

As pointed out by Ryo Kataoka.
(cherry picked from commit 05c38544b857d0c7c9735713b7dd8ea396c13c23)

Conflicts:
	arch/arm/mach-shmobile/Kconfig

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 arch/arm/mach-shmobile/Kconfig | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm/mach-shmobile/Kconfig b/arch/arm/mach-shmobile/Kconfig
index efa055b..0896f4a 100644
--- a/arch/arm/mach-shmobile/Kconfig
+++ b/arch/arm/mach-shmobile/Kconfig
@@ -39,13 +39,6 @@ config ARCH_R8A7790
 	select SH_CLK_CPG
 	select RENESAS_IRQC
 
-config ARCH_R8A7790
-	bool "R-Car H2 (R8A77900)"
-	select ARM_GIC
-	select CPU_V7
-	select ARM_ARCH_TIMER
-	select SH_CLK_CPG
-
 config ARCH_R8A7791
 	bool "R-Car M2 (R8A77910)"
 	select ARM_GIC
-- 
1.8.3.2

